worlds/tunic/options.py

```diff
@@ -189,6 +168,82 @@ class TunicPlandoConnections(PlandoConnections):
     duplicate_exits = True
 
 
+class LaurelsZips(Toggle):
+    """
+    Choose whether to include using the Hero's Laurels to zip through gates, doors, and tricky spots.
+    Notable inclusions are the Monastery gate, Ruined Passage door, Old House gate, Forest Grave Path gate, and getting from the Back of Swamp to the Middle of Swamp.
+    """
+    internal_name = "laurels_zips"
+    display_name = "Laurels Zips Logic"
+
+
+class IceGrappling(Choice):
+    """
+    Choose whether grappling frozen enemies is in logic.
+    Easy includes ice grappling enemies that are in range without luring them. May include clips through terrain.
+    Medium includes using ice grapples to push enemies through doors or off ledges without luring them. Also includes bringing an enemy over to the Temple Door to grapple through it.
+    Hard includes luring or grappling enemies to get to where you want to go.
+    The Medium and Hard options will give the player the Torch to return to the Overworld checkpoint to avoid softlocks. Using the Torch is considered in logic.
+    Note: You will still be expected to ice grapple to the slime in East Forest from below with this option off.
+    """
+    internal_name = "ice_grappling"
+    display_name = "Ice Grapple Logic"
+    option_off = 0
+    option_easy = 1
+    option_medium = 2
+    option_hard = 3
+    default = 0
+
+
+class LadderStorage(Choice):
```

Comment:
I love that this can be included in logic lol

Reply:
hahaha it's a nightmare